src/commit/mult.ts

```diff
@@ -94,40 +74,33 @@ export async function proveMult(
     params: PedersenParams,
     x: bigint,
     y: bigint,
-    z: bigint,
+    _z: bigint,
     Cx: Commitment,
     Cy: Commitment,
     Cz: Commitment
 ): Promise<MultProof> {
     const xx = params.c.newScalar(x),
-        C4 = Cy.p.mul(xx), // C4 = Cy * x
-        r4 = Cy.r.mul(xx), // C4 = zG + r4H
         // Step 1: Compute commitments
         k_x = rnd(params.c.order),
         k_y = rnd(params.c.order),
         k_z = rnd(params.c.order),
-        kx = params.c.newScalar(k_x),
+        ky = params.c.newScalar(k_y),
```

Comment:
To make my life I easy I would appreciate completely changing the notation to match that of the paper. It's very difficult to determine which variables are supposed to mean which without doing that.